examples/6-advanced-flows.html

```diff
@@ -0,0 +1,213 @@
+    // that maps the first thing the bot will say to the user
+        ice: {
+        // "says" defines an array of sequential bubbles
+        // that the bot will produce
+        says: [
+            "Hello",
+            "We're exploring chat flows.",
+            "<img src=https://i0.wp.com/meanbusiness.com/wp-content/uploads/2018/03/Coral-Share_Ideas-350x198.gif?fit=300 />",
+            "Do you have 90 seconds to spend with us?",
+            "(You can win Prizes!) [Sidetrack to the main flow]",
+        ],
+        // "reply" is an array of possible options the user can pick from
+        // as a reply
+      reply: [
+            {
+        question: "Sure", // label for the reply option
+    answer: "chapter-one" // key for the next conversation object
+            },
+            {
+        question: "No. Thanks [Escape Valve hit]", // label for the reply option
+    answer: "end" // key for an Escape Valve. We refer to this whenever a reply signals the end of the convo. 
+            },
+            {
+        question: "What Kind of Prizes? [Sidetrack hit]", // label for the reply option
+    answer: "sidetrack" // key for a side note we can reference from multiple points in the chat
+            }
+        ]
+    }, // end required "ice" conversation object
+    "sidetrack": {
+        "says": [
+            "Fabulous prizes!*",
+            "<img src=https://media.giphy.com/media/cPUMQkF2A19vy/giphy.gif>",
+            "<span style='color: #999; font-size: 13px;'>(* <em>Prize Fabulosity is not guaranteed.</em></span>",
+            "<span style='color: #999; font-size: 13px;'><em>Use only as directed. Keep away from sunlight, pets, and small children.</em></span>",
+            "<span style='color: #999; font-size: 13px;'><em>No purchase necessary. Allow four to six weeks for delivery.</em></span>",
+            "<span style='color: #999; font-size: 13px;'><em>Toy does not actually fly.</em>)</span>",
+        ],
+        "reply": [
+            {
+                "question": "Great. So what are the basics? [Give me context]",
+                "answer": "intro-context" // key for another side note. In this instance, we're using it to contextualize the Q&A we're heading into 
+            },
+            {
+                "question": "Got it. Play On!", 
+                "answer": "chapter-one" // No further sidetrack required
+            }
+        ]
+    },
+    "intro-context": {
+        "says": [
+            "We give you questions and choices --",
+            "[And other info about the ensuing dialogue]",
+            "You tap or click the best responses."
+        ],
+        "reply": [
+            {
+                "question": "Play On!",
+                "answer": "chapter-one"
+            }
+        ]
+    },
+    "chapter-one": {
+        "says": [
+            "Excellent -- Thank you.",
+            "[Questions | Info | Requests for feedback]",
+        ],
+        "reply": [
+            {
```

Comment:
This set (and the following few) of answers is a little confusing. I understand this is an example, but perhaps we could make up some imaginary questions and answers to match closer to an actual use-case?

Reply:
I can keep the narrative to Bananas and Ice Cream for continuity with your other examples. 

The chat is based on [an example use case](https://meanbusiness.com/gamepath/chat-bubble/examples/Coral-intro-chat.html) I did for the [Coral Project](https://www.coralproject.net/) to represent personality types in News communities.

Reply:
This is awesome, love the project! How about using the bananas & ice cream for now; but would you like to add a link to your project to the main Readme.md?